### ISSUE TITLE
sometime reboot win10 guest and  change funtion will 43 error code

### ISSUE THREAD

**Issue by suancainiuhuanxi** [strongtz/i915-sriov-dkms#131]
echo 7 > /sys/devices/pci0000:00/0000:00:02.0/sriov_numvfs
create 7 vgpu and i give the 0000:00:02.1 to vm win10 ,  it can work success , sometime reboot also will show the driver 43 code, sometime change it to 0000:00:02.2, it happen the intel driver code 43 error.  and sometime reboot also will show the driver 43 code 
and i create 4 vgpu it aslo happen
my host os is ubuntu23.04
kernel is 6.2.0-20-generic,
libvirt version: 9.0.0
intel cpu: 13th Gen Intel(R) CoreT i9-13900H
my win10 driver : Graphics Driver 31.0.101.4953

Has anyone been in a similar situation？

**Comment by scyto** [comment]
yes - this seems to happen a few reboots after windows hello is enabled in the VM (joining to AAD or using a microsoft account)
